api/filters.go

```diff
@@ -940,7 +943,7 @@ func (api *FilterAPI) getOutput(ctx context.Context, filterID string) (*models.F
 	errFilterOutputNotFound := errors.New("Filter output not found")
 
 	// Hide private download links if request is not authenticated
-	if !common.IsCallerPresent(ctx) {
+	if r.Header.Get(common.DownloadServiceHeaderKey) != api.downloadServiceToken {
```

Comment:
Slightly pedantic but would it be worth logging that we are removing / allowing the links?

Reply:
done 👍 - also improved the existing logging slightly.
 - reuse common logData
 - declare error at package level